kong/router/atc.lua

```diff
@@ -468,7 +474,7 @@ function _M:exec(ctx)
   local fields = self.fields
 
   local req_uri = ctx and ctx.request_uri or var.request_uri
-  local req_host = var.http_host
+  local req_host = get_header("host", ctx)
```

Comment:
I think that here we can use `var.http_host` safely, because multiple host header is invalid.

Reply:
but var access is probably a bit slower than retrieving value from already cached lua table ctx.